src/components/Chat/InputMessageV2.tsx

```diff
@@ -388,7 +388,7 @@ const InputMessageV2 = ({
         '',
         type === ANONYMOUS
           ? `Switch back to regular chat?\nMessage ${username} using your username.`
-          : 'Switch to anonymous chat?\nMessage this user anonymously instead.',
+          : 'Switch to anonymous chat?\nMessage this user in Incognito Mode instead.',
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The change in the message text from 'Message this user anonymously instead.' to 'Message this user in Incognito Mode instead.' is a good one as it provides more clarity to the user about the mode of chat they are switching to. However, ensure that the term 'Incognito Mode' is clearly defined and understood by your users elsewhere in your application.
```diff
- 'Switch to anonymous chat?\nMessage this user anonymously instead.',
+ 'Switch to anonymous chat?\nMessage this user in Incognito Mode instead.',
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->